gobblin-metrics/src/test/java/gobblin/metrics/performance/MetricsPerformanceTest.java

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright (C) 2014-2015 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.metrics.performance;
+
+import lombok.Builder;
+import lombok.Singular;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.Comparator;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeMap;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+import org.testng.annotations.Test;
+
+import com.beust.jcommander.internal.Lists;
+import com.google.common.collect.Maps;
+import com.google.common.collect.Sets;
+
+
+/**
+ * Class for running Gobblin metrics performance tests.
+ */
+@Slf4j
+public class MetricsPerformanceTest {
+
+  @Test(groups = { "performance" })
```

Comment:
Can't you add the `groups = { "performance" }` on a class level so that you don't have to do it for each method?